scripts/bundleSize.js

```diff
@@ -0,0 +1,126 @@
+#! /usr/bin/env node
+
+const ora = require('ora');
+const { execSync } = require('child_process');
+const chalk = require('chalk');
+
+const services = Object.keys(
+  require('../src/app/lib/config/services/loadableConfig').default,
+);
+
+// Size limit for all bundles used by each service (K)
+// Keep these +/- 5K and update frequently!
+const MIN = 557;
+const MAX = 576;
+
+const getBundleSize = filePattern => {
+  const size = execSync(`cat ${filePattern} | wc -c | tr -d ' '`, {
```

Comment:
Could we maybe use `fs` for this? If we run something like this would avoid `execSync` which is preferable IMO, thoughts?
```js
const fs = require('fs');

const sizeData = {};
const dir = 'build/public/static/js';
const jsFiles = fs.readdirSync(dir).filter(path => path.endsWith('.js'));

jsFiles.forEach(path => {
  sizeData[path] = fs.statSync(`${dir}/${path}`).size;
});

// match against `Object.keys(sizeData)` to get sizes
```
^ feel free to do it in a different way if you think preferable :)

P.s. super thanks for this PR 🙈

Reply:
👍 done